vector-im/element-android
# [Poll] Render active polls list of a room (PSG-908)

## Type of change


## Content


Start of the implementation for poll history of a room. Adding a new entry in the room profile screen (only available in debug build for now). Render a mocked list of active polls.

## Motivation and context


Part  of #7864 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/210081872-7c38d611-9469-4395-8213-6685522e3233.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/210081876-947928a7-69ac-4f2c-87da-f04b46237c6f.png" width=300 />

<img src ="https://user-images.githubusercontent.com/46314705/210804274-94e7f62b-50f9-4087-962d-0e195d2d5637.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/210804281-5c358449-e03e-412d-af27-149293600df6.png" width=300 />

<img src="https://user-images.githubusercontent.com/46314705/210082159-5d51c5b7-0c07-442e-9cf6-2dc5119d3a83.png" width=300 /> <img src ="https://user-images.githubusercontent.com/46314705/210082156-866b58cc-ddc8-401d-bb82-436acfdc19fc.png" width=300 />

## Tests



- Go to the profile screen of a room
- Press the "Poll history" entry
- Check there is tab displaying a mocked list of active polls

## Tested devices

- OS version(s): Android 11

## Checklist
